### PR TITLE
fix unbound error

### PR DESCRIPTION
**Pull request by masco**

### CODE REVIEW

**Review by amitsagtani97 — APPROVED** [review]
LGTM!

File: jjb/dynamic/scale-ci_install_osp.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@
         do
         
         # Get the day of week and set networkType as OpenshiftSDN on odd days and kuryr on even days in case no network is defined manually.
-        if [ $OPENSHIFT_NETWORK_TYPE == "" ]
+        if [ "$OPENSHIFT_NETWORK_TYPE" == "" ]
```
Comment (chaitanyaenr) [inline jjb/dynamic/scale-ci_install_osp.yml:26]: Maybe use -z flag for the check? Thoughts?

Comment (masco) [reply jjb/dynamic/scale-ci_install_osp.yml]: totally agree, done!

**Review by chaitanyaenr — APPROVED** [review]
LGTM

**Pull request merged**